README.md

```diff
@@ -1,8 +1,6 @@
 # conjur-api-python3
 
-Python3-based API SDK for [Conjur OSS](https://www.conjur.org/). The repo
-also includes a self-contained CLI tool (`conjur-cli`) that wraps the API
-in a simple executable script/binary.
+Python3-based API SDK for [Conjur OSS](https://www.conjur.org/). The repo also includes a self-contained CLI tool (`conjur`) that wraps the API in a simple executable script/binary.
```

Comment:
Many `line-too-long`s in this file too

Reply:
yes Geri also commented this and because I am using Typora I cannot really manipulate the line length. Once I finish here I will go into the MD and correct this